modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -1,6 +1,49 @@
-generate_joint_ensemble_design <- function() {
- inputsamples<-input.ens.gen()
- paramsample<-get.parameter.samples()
+generate_joint_ensemble_design <- function(settings, ensemble_size, 
+                                           posterior.files = rep(NA, length(settings$pfts)), 
+                                           ens.sample.method = "uniform") {
```

Comment:
Does this have to be an argument? can't this come from the settings like it it is for input.ens.gen?